CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+## Contributing
+Standard PR workflow
+
+## Structure of the testsuite
+Group all tests for given MP spec into single module
+* for example to group all tests for MP Metric together
+* in case that tests for integration of multiple MP specs are developed and it's not clear to which module they belong then new module can be created. Preferable a name of such module should reflect MP specs.
+
+Tooling should be in separate module so it is isolated and easily used by other (test) modules. Try to add tooling to existing module. If there is no module that reflect tooling purpose you can create new module.
+
+Rather than add new dependency try to extend existing tooling or add new one yet still consider the trade-off.
+
+Examples of available tooling:
+* tooling-cpu-load
+* tooling-docker
+* tooling-server-configuration
+
+##### tooling-server-configuration
+Module contains classes for configuring and managing server. To do so you ought to use:
+* `MicroProfileServerSetupTask` as `ServerSetupTask`
+* Creaper core as an abstraction to `ModelNode` operations
```

Comment:
There is same thing described across next 3 bullets. IMHO the best solution would be to provide a link to module's README where it is better explained how to use described resources.

> * Creaper core as an abstraction to `ModelNode` operations
> * `OnlineManagementClient` provided by Creaper core

This might seem great for us, but it IMHO says nothing useful to general audience. What is Creaper core and what it is useful for? What is `ModelNode` (well that might be actually known by the audience)? Something like which could be a starting point for linking to relevant part of module's README:

> An abstraction using Creaper core library to provide means for server configuration.

WDYT?